Library/Formula/silk.rb

```diff
@@ -1,9 +1,7 @@
-require "formula"
-
 class Silk < Formula
-  homepage "http://tools.netsa.cert.org/silk/"
-  url "http://tools.netsa.cert.org/releases/silk-3.9.0.tar.gz"
-  sha1 "40dd51401f8688cf5e9a9b0146e1a9e0ccb654aa"
+  homepage "https://tools.netsa.cert.org/silk/"
+  url "https://tools.netsa.cert.org/releases/silk-3.10.0.tar.gz"
+  sha256 "935d089479f0b51d6408a6afc497ab51a69751cf2b473a652ecb3b16465e0f5a"
```

Comment:
I don't think this needs bumped to `sha256` unless it's displayed more prominently on their homepage.